packages/devops/scripts/lambda/deleteDeployment.py

```diff
@@ -4,46 +4,37 @@
 import re
 import asyncio
 import functools
+from utilities import *
 
 ec2 = boto3.resource('ec2')
 ec = boto3.client('ec2')
 iam = boto3.client('iam')
 route53 = boto3.client('route53')
+elbv2 = boto3.client('elbv2')
 
-def get_tag(instance, tag_name):
-    try:
-        tag_value = [
-            t.get('Value') for t in instance['Tags']
-            if t['Key'] == tag_name][0]
-    except IndexError:
-        tag_value = ''
-    return tag_value
-
-def get_instances(filters):
-    reservations = ec.describe_instances(
-        Filters=filters
-    ).get(
-        'Reservations', []
+
+def delete_gateway(tupaia_instance_name):
+    gateway_elb = get_gateway_elb(tupaia_instance_name)
```

Comment:
Ah right, I see why you've got the util allowing lookup by tag name now. Could save the variable earlier in the main script and use it again here though

Reply:
Discussed, gateway may not exist due to being different resources

Reply:
Discussed, separate resources, have to look up by tags